ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -90,29 +90,44 @@ import SwiftUI
 }
 
 /// Properties available to customize the state of the avatar
-@objc public class MSFAvatarState: NSObject, ObservableObject {
-    @objc @Published public var image: UIImage?
-    @objc @Published public var primaryText: String?
-    @objc @Published public var secondaryText: String?
-    @objc @Published public var ringColor: UIColor?
-    @objc @Published public var backgroundColor: UIColor?
-    @objc @Published public var foregroundColor: UIColor?
-    @objc @Published public var presence: MSFAvatarPresence = .none
-    @objc @Published public var isRingVisible: Bool = false
-    @objc @Published public var isTransparent: Bool = true
-    @objc @Published public var isOutOfOffice: Bool = false
+@objc public protocol MSFAvatarState {
```

Comment:
Can you update with the latest commit here? The iPad pointer interaction support property will need to be here too.